config/default-amo.js

```diff
@@ -25,9 +25,34 @@ module.exports = {
   },
   enableTrailingSlashesMiddleware: true,
   fxaConfig: 'amo',
+
+  // This needs to be kept in sync with addons-server's SUPPORTED_NONLOCALES
+  // settings value: https://github.com/mozilla/addons-server/blob/master/src/olympia/lib/settings_base.py#L274
+  // These are URLs that are ignored by our prefix middleware that will add
+  // a locale (e.g. `en-US`) to any URL that doesn't have a valid locale.
+  // These are all URLs that should not prepend the locale to the URL if it's
```

Comment:
Thanks for adding extra comments. I think this sentence needs to be reworded though. "These are all URLs that should not get a locale prepended ... " ?